migration/v2.18.0-ck8s1-v2.19.x-ck8s1/upgrade-cluster.md

```diff
@@ -14,6 +16,43 @@
     imageGCLowThresholdPercent: 70
     ```
 
+1. add the following snippet at the end of both `sc-config/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml` and `wc-config/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml`
```

Comment:
Should you also add a note about changing `etcd_kubeadm_enabled: true` to `etcd_deployment_type: kubeadm`?

Reply:
Yes will do that, have some other things i got to update aswell.